src/lib/components/delegations/DelegationInfo.tsx

```diff
@@ -43,16 +47,8 @@ export const DelegationInfo = ({
           <CustomIcon name="chevron-right" color="gray.600" />
         </Flex>
       ) : (
-        <Flex direction="column" gap={4}>
-          <Heading
-            variant="h6"
-            as="h6"
-            display="flex"
-            alignItems="center"
-            height={{ base: "29px", md: "auto" }}
-          >
-            Delegations
-          </Heading>
+        <Flex direction="column" gap={4} mt={{ base: 4, md: 0 }}>
+          {tableTitle}
```

Comment:
can we just use `<TableTitle title="Delegations" mb={0} showCount={false} />` here?